wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests.wiki

```diff
@@ -123,7 +123,7 @@ The [[heuristic to find places][>FindingByHeuristics]] tries a number of transla
 |switch to frame <technical selector>                                      |Activates child frame of current (i)frame found using 'technical selector'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
 |switch to parent frame                                                    |Activates parent frame of current (i)frame. Does nothing if when current frame is the main/top-level one.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 |set implicit find in frames to <true/false>                               |Controls whether !-BrowserTest-! should look for elements inside nested (i)frames if they cannot be found in the current frame (default is 'true').                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-
+|value of <place> is valid                                                 |Checks if entered value in input field is valid according the pattern attribute. If 'place' doesn't resolve to an element, an exception is thrown. If the field doesn't has both the id and name attributes set, an exception is thrown.                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
Please change description to 'Checks whether entered value in input field ('place') is valid according to its pattern attribute.'

Reply:
Ok